src/platform/plugins/shared/inspector/public/ui/inspector_panel.tsx

```diff
@@ -134,15 +134,15 @@ export class InspectorPanel extends Component<InspectorPanelProps, InspectorPane
                 <h1>{title}</h1>
               </EuiTitle>
             </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <InspectorViewChooser
-                views={views}
-                onViewSelected={this.onViewSelected}
-                selectedView={selectedView}
-              />
-            </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlyoutHeader>
+        <EuiFlexItem className="insInspectorViewChooserContainer">
```

Comment:
Is `EuiFlexItem` still needed, since this is not longer in `EuiFlexGroup`. how about just using a `div` here.